platforms/netlify-ci-build/README.md

```diff
@@ -1,7 +1,8 @@
 # Introduction
 
-Prisma and Netlify integration
+Prisma and Netlify (beta build) integration.
```

Comment:
Can we link somewhere so one can know what "beta build" refers to?

Reply:
Yes, leaving a note for myself here: https://build-beta.netlify.com/